### DISCUSSION TITLE
Does hextra theme support asciidoc as a doc format?

### DISCUSSION THREAD

**Discussion by mmurdockk** [imfing/hextra#391] [Q&A] [answered]
Hey there, I try to move from markdown to asciidoc and have some issues related to tableofcontent etc.
Is it something supported by hextra?

Thank you

**Comment by imfing** [comment, marked as answer]
Hi, this project is currently developed mainly around Markdown format. While Hugo supports asciidoc, I didn't test with it so it is expected that things may break.
